analyzer/habitat/analysis/operation.py

```diff
@@ -81,6 +81,7 @@ def device(self):
 
     def to_device(self, dest_device, predictor):
         if dest_device.name == self._device.name:
+            warnings.warn("Predicting to the same device")
             return self
```

Comment:
Remember to delete this line so it falls through

Reply:
done